src/scene_battle_rpg2k.cpp

```diff
@@ -374,6 +373,9 @@ void Scene_Battle_Rpg2k::ProcessActions() {
 bool Scene_Battle_Rpg2k::ProcessBattleAction(Game_BattleAlgorithm::AlgorithmBase* action) {
 	// Order of execution of BattleActionState:
 	// ConditionHeal > Execute > Apply > (ResultPop > ResultPush) > Death > Finished.
+	if (action == nullptr) {
```

Comment:
Is it possible that the action is nullptr?
If not add a noise Output::Warning here so we get bugreports.

Reply:
Thats an artifact I forgot to remove from an old implementation of this I tried.

Reply:
okay then just remove it